common/config/config.go

```diff
@@ -116,6 +116,8 @@ type (
 		Frontend GroupTLS `yaml:"frontend"`
 		// SystemWorker controls TLS setting for System Workers connecting to Frontend.
 		SystemWorker WorkerTLS `yaml:"systemWorker"`
+		// RemoteFrontendClients controls TLS setting for talking to remote cluster.
+		RemoteClusters map[string]GroupTLS `yaml:"remoteClusters"`
```

Comment:
how does this work with the dynamic added remote cluster via api?

Reply:
My understanding is that we will use the same process as we do today to periodically sync certs and injects them into config from a known file location on host. There will be a separate process to sync the cert file from the central cert management system to that file location as well.
Neither is the concern for OSS based on my understanding.
@sergeybykov can correct me if I am wrong

Reply:
Correct - we do update TLS config dynamically when needed.